.env.example

```
@@ -61,6 +61,20 @@ ADMINS=Admin Name:admin@exmaple.com,Admin2 Name:admin2@exmaple.com
+# Extra uwsgi flags appended to the web container's `uwsgi --ini` command (later
+# flags win). Leave unset for the in-file defaults (tuned for c5a.large). When
+# scaling to a bigger instance for the September load spike, raise the worker
+# ceiling here instead of editing uwsgi.aws.ini -- see production/SEPTEMBER-RUNBOOK.md.
+#UWSGI_EXTRA_ARGS=--processes 12
+
+# Redis memory ceiling (noeviction). Defaults to 512mb. Hitting the cap makes
+# redis refuse all writes (breaking the celery broker), so size it to the
+# instance + working set; raise it (and give the box RAM) before a load spike.
+#REDIS_MAXMEMORY=512mb
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)